base_layer/wallet_ffi/tests/test_error_scenarios.py

```
@@ -0,0 +1,890 @@
+            try:
+                from tari_wallet import SimpleDiscoveryService, TariNetwork
+                discovery = SimpleDiscoveryService(TariNetwork.NEXTNET)
+                
+                # Test invalid timeouts
+                invalid_timeouts = [-1, 0, float('inf')]
+                handled_timeouts = 0
+                
+                for timeout in invalid_timeouts:
+                    try:
+                        discovery.discover_and_select_node(dns_timeout=timeout)
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)